caluma/caluma_form/factories.py

```diff
@@ -196,9 +196,11 @@ def files(self, create, extracted, **kwargs):
                 self.files.set(
                     FileFactory.create_batch(num_files, **kwargs, answer=self)
                 )
+            self.save()
```

Comment:
Is this really needed? `self.files` is a M:N relationship, and the intermediate table should be written directly.